src/lib/components/WellCompletions/components/Settings/WellAttributesSelector.test.tsx

```diff
@@ -18,13 +21,14 @@ describe("test well attributes selector", () => {
         render(<WellAttributesSelector />, {
             wrapper: Wrapper,
         });
-        
-        const attributeFilter = screen.getByRole('textbox')
-        userEvent.type(attributeFilter, 'type:Injector{enter}')
-        await waitFor(() => expect(screen.getByText(/well selection criteria: "type" is "injector"/i)).toBeVisible())
```

Comment:
This is not going to happen because it requires the state to be updated in the redux store to be able to see the criteria text update, but this testStore.dispatch is just a mocked-up function.